go/discovery/aggregated.go

```diff
@@ -99,12 +100,18 @@ func aggregate(results []collection.Metric) AggregatedDiscoveryMetrics {
 			last = v.Timestamp
 		}
 
-		// successful names
+		// different names
 		x := names["InstanceKeys"]
 		x[v.InstanceKey.String()] = 1 // Value doesn't matter
 		names["InstanceKeys"] = x
-		// failed names
-		if v.Err != nil {
+
+		if v.Err == nil {
+			// ok names
+			x := names["OkInstanceKeys"]
```

Comment:
Can you please make all these strings a type? See for example [AnalysisCode](https://github.com/github/orchestrator/blob/0333dfa6c37bd5c14fd4e9f13cb0a841d1299745/go/inst/analysis.go#L23-L54)